Common/UnitDefinitions/Pressure.json

```diff
@@ -354,14 +354,19 @@
       ]
     },
     {
-      "SingularName": "MillimeterOfWaterColumn",
-      "PluralName": "MillimeterOfWaterColumn",
-      "FromUnitToBaseFunc": "{x} * 9.806650000000272e0",
-      "FromBaseToUnitFunc": "{x} / 9.806650000000272e0",
+      "SingularName": "MeterOfWaterColumn",
+      "PluralName": "MeterOfWaterColumn",
+      "FromUnitToBaseFunc": "{x} * 9.806650000000272e3",
+      "FromBaseToUnitFunc": "{x} / 9.806650000000272e3",
+      "Prefixes": [ "Milli", "Centi" ],
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "mmH2O", "mm wc" ]
+          "Abbreviations": [ "mH₂O", "mH2O", "m wc", "m wg" ],
+          "AbbreviationsForPrefixes": { 
+            "Milli": [ "mmH₂O", "mmH2O", "mm wc", "mm wg" ], 
+            "Centi": [ "cmH₂O", "cmH2O", "cm wc", "cm wg" ]
```

Comment:
I think you can remove `AbbreviationsForPrefixes` unless you need to customize it, it will by default add `c` and `m` to each unit abbreviation.

Reply:
Thanks, it is a cool feature! Fixed.